.github/workflows/ci.yml

```diff
@@ -5,7 +5,21 @@ on:
   pull_request:
 
 jobs:
+  test-repository:
+    runs-on: ubuntu-latest
+    env:
+      NPM_CONFIG_WORKSPACE: repository
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+      - uses: supercharge/redis-github-action@1.8.0
+      - run: npm ci
+      - run: npm test
```

Comment:
For my understanding, what does root `npm test` test that workspace `npm test` doesn't? Is it the README?

Reply:
The root `package.json` "test" command runs also the linter and TypeScript.

```
    "test": "npm run lint && npm run test:types && npm run test:unit"    
```

I am using the same `ci.yaml` structure as we have in spark-stats, I just renamed "build-" prefix to "test-" prefix, as I find it more accurate.

https://github.com/filecoin-station/spark-stats/blob/8a3ec03142d5268b96d2846cf2d81efea7c158bc/.github/workflows/ci.yml